docs/devel/releasing.md

```diff
@@ -144,7 +144,7 @@ Decide what version you're cutting and export it:
 Then, run
 
 ```console
-./release/cut-official-release.sh "${VER}" "${GITHASH}"
+./release/cut-official-release.sh "${VER}" "${GIT_COMMIT_HASH}"
```

Comment:
@thockin this is the only place the exported variable is used as far as I can tell.